net-gateway/src/endpoints/networks/add_network.rs

```diff
@@ -66,35 +63,16 @@ async fn network(
 
     match server_connection.send_all_reliable(&request.encode()).await {
         Ok(_) => (),
-        Err(e) => return HttpResponse::InternalServerError().json(
-            serde_json::json!({
-                "error": "Internal Server Error",
-                "message": format!("Cound't insert the data: {}", e.to_string()),
-            })
-        ),
+        Err(_) => return Err(UserFacingError::Timeout),
```

Comment:
Should probably be `unwrap_or` here, instead of this. And, like... Everywhere

Reply:
I will fix this later

Reply:
We must return something from endpoint anyway